### ISSUE TITLE
Engineering

### ISSUE THREAD

**Issue by Raithaam** [b-morgan/Skillet-Classic#1]
Maybe there is a simple fix I'm missing, but my engineering recipes do not show up. It shows the category headings and the number of recipes in each, but the recipes themselves are not there at all except for the Inlaid Mithril Cylinder Plans.  Note this started with the new Classic version it was fine with the old version

![Skillet](https://user-images.githubusercontent.com/77211907/110036782-64ccb800-7cf2-11eb-9011-ffbf2c065def.jpg)

**Comment by b-morgan** [comment]
In order to figure out what is wrong, I would need your global saved variables file, Skillet-Classic.lua. You will have to zip it to attach it here. If you prefer, just delete that file and I'm guessing it will fix itself.

**Comment by Raithaam** [comment]
So, I'm not seeing a "Skillet-Classic.lua" at all. I'm in the interface/addons/skillet-classic folder. There is a Skillet-Classic.toc. I have tried uninstalling the addon completely and reinstalling it several times.

**Comment by b-morgan** [comment]
World of Warcraft addons have the source code stored in `...\World of Warcraft\_classic_\Interface\Addons\<plugin name>` and global data in `...\World of Warcraft\_classic_\WTF\Account\<user name>\SavedVariables\<plugin name>`. Per character data is in `...\World of Warcraft\_classic_\WTF\Account\<user name>\<realm name>\<character name>\SavedVariables\<plugin name>` (Note: `_classic_` is replaced by `_retail_` for the original World of Warcraft).

So now that you know where to look, please zip your global saved variables file Skillet-Classic.lua and upload it here. Then delete your global saved variables file Skillet-Classic.lua and Skillet-Classic.lua.bak (if it exists) while World of Warcraft is closed. Logon and see if the problem still exists. Let me know either way (fixed or not fixed). 

If I ask for both global and per character data, then rename the per character file to be <character name>.lua and you can zip them both together. The per character data will contain debugging information from Skillet-Classic if we have to turn that on.

**Comment by Raithaam** [comment]
I went to the ...\World of Warcraft\_classic_\WTF\Account\<user name>\SavedVariables\<plugin name> and didn't see Skillet there at all. I thought maybe I hadn't reinstalled it last thing last night, so I checked and it was installed. Opened the game and it was working fine. Went back and I realized I had two different folders for the same account in the WTF folder. I assume this is left over from when I moved my second account onto the same Battlenet account as my primary. Anyway, I deleted the older one that didn't have more recent addon data in it. The problem seems to be resolved. Hope I didn't waste your time with this!

Fixed.
